source/triggers/database-triggers.txt

```diff
@@ -53,6 +43,27 @@ expressions to limit the data included in each event.
    <serverless-caveats>` or :ref:`{+adf-instance+}
    <data-federation-caveats>` because they do not support change streams.
 
+You control which operations cause a trigger to fire as well as what
+happens when it does. For example, you can run a function whenever a
+specific field of a document is updated. The function can access the
+entire change event, so you always know what changed. You can also pass
+the change event to :ref:`AWS EventBridge <aws-eventbridge>` to handle
+the event outside of Atlas.
+
+Triggers support :manual:`$match </reference/operator/aggregation/match>` 
+expressions to filter change events and :manual:`$project </reference/operator/aggregation/project>` 
+expressions to limit the data included in each event.
+
+.. _trigger-recursion:
+
+.. warning::
+
+   It is possible to configure triggers that cause other triggers to fire on the 
+   same collection, database, or cluster, resulting in recursion. Examples 
+   include a database trigger writing to a collection within the same 
+   database, or a cluster-level logger writing database logs to another database 
+   in the same cluster.
+   
```

Comment:
I think we may want to explicitly call out log forwarders as well here. Since, they are a first class resource that our users may also be using that could cause this issue.

Reply:
Isn't that just a specific case of a process writing to a monitored database and/or collection?

Reply:
You are correct, but just wanted to make customer's slightly more conscious if they didn't make that connection.

Reply:
Talked offline in the project channel with team and Laura, we would like to go with.

```suggestion
.. warning::

   In deployment and database level triggers, it is possible to configure triggers 
   in a way that causes other triggers to fire, thereby resulting in recursion. 
   Examples include a database-level trigger writing to a collection within the 
   same database, or a cluster-level logger or log forwarder writing logs to 
   another database in the same cluster.
   
```

Reply:
@MongoCaleb Could we get an update on this since it should be going in tomorrow.